apps/admin/pages/api/auth/[...nextauth].ts

```diff
@@ -2,10 +2,12 @@ import NextAuth from "next-auth";
 import GoogleProvider from "next-auth/providers/google";
 import { PrismaAdapter } from "next-auth-prisma-adapter";
 import prismaClient from "db";
+import CredentialsProvider from "next-auth/providers/credentials";
 
 export default async function auth(req: any, res: any) {
   return await NextAuth(req, res, {
     //debug: true,
+    secret: "LlKq6ZtYbr+hTC073mAmAh9/h2HwMfsFo4hrfCx5mLg=",
```

Comment:
PR can be "at the most" a draft with this in place

Reply:
It's copied from nextauth, just a random string, so it doesn't throw an error